code/modules/reqs/supplypacks.dm

```diff
@@ -670,6 +670,7 @@ ARMOR
 	contains = list(
 		/obj/item/clothing/suit/storage/marine/specialist,
 		/obj/item/clothing/head/helmet/marine/specialist,
+		/obj/item/clothing/gloves/marine/specialist,
```

Comment:
b17 doenst has these too ?

Reply:
yeah cause this is B18 gloves (which isn't named B18 for some reason)

Reply:
no i mean, doenst b17 have a B17 gauntlets as well ? funky

Reply:
> 
> 
> no i mean, doenst b17 have a B17 gauntlets as well ? funky

not that i can see in the code; its just B18